src/components/common.js

```diff
@@ -275,6 +275,16 @@ export default {
       }
 
       return this.id;
+    },
+    zarrMetadataUrl() {
+      const zarrMetadata = this.assets.find(x =>
+        x.roles.includes("zarr-consolidated-metadata")
+      );
+
+      if (zarrMetadata != null) {
```

Comment:
Arra.find returns undefined if no result is found, not null.

```suggestion
      if (typeof zarrMetadata !== 'undefined') {
```